packages/aws-cdk-lib/aws-codepipeline-actions/lib/codecommit/source-action.ts

```diff
@@ -54,6 +55,35 @@ export interface CodeCommitSourceVariables {
   readonly commitMessage: string;
 }
 
+/**
+ * Represents a custom event rule in AWS CodePipeline Actions.
+ *
+ * This interface defines the structure for specifying a custom event rule
+ * in the AWS CodePipeline Actions module. The event rule is defined by an
+ * event pattern and a target.
+ *
+ * @see https://docs.aws.amazon.com/codecommit/latest/userguide/monitoring-events.html
+ * @see https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_events_targets-readme.html
+ */
+export interface ICustomEventRule {
+  /**
+   * event pattern when this rule should be triggered 
```

Comment:
```suggestion
   * event pattern when this rule should be triggered
```

Reply:
@jolo-dev looks like it was just an extra trailing space causing the linter to fail. I'll commit this change in. Usually when build fails, I'll open up the [Build Logs](https://ops-cdk-autobuild-prs-autobuildv2github-buildlogs-g9jsj1nfmnjp.s3.amazonaws.com/c9d13ee1-58f2-44cd-a0a7-1e49a2d33d44/build.log?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=ASIAWYORDRBKPVC4YM5S%2F20240115%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20240115T221235Z&X-Amz-Expires=600&X-Amz-SignedHeaders=host&X-Amz-Security-Token=IQoJb3JpZ2luX2VjEO%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2FwEaCXVzLWVhc3QtMSJHMEUCIQC3cQwceeXeKl7j0KvbcaOKeI1TPeA9aWOsOrI8uvWLaAIgXBBqEUhHSlcD3C5TyYIlntVfDjFCnbBJaS56AD8brIcq3AMIl%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2FARACGgw0NjQ4MzE4NzUxNTYiDCRMRDcQe7f0%2BwMzkyqwA3Oj%2B7D2pQ4V7Z14T3PHK%2BtBcBB%2BlbPPdWhnm16zc3fBKi0Wc%2F3eUT1fOJ2NClWja1woDgmzusIpLdoQF9A%2B3JesCK4KytiNBiknx1615mqNzhUcLHb4P5J2H1XlX7%2F25L660G3JLMRKvclnpgNLH5hllBKj5ep%2BIUGmkUNMwxE5oN%2F%2FAi4gx4rf7LeLXI3iywrIxY%2BmnlVn%2FFKIm4ToAGvSOP4Mixwl%2F1B2rwSXCDXMwRMM56Zcd7pBggUDLuKW9haCKydbEGMORBZwE7NPWMzZI4BSdO31%2BjwIf8mfqsNvSwwK7Hy%2BG4zbnDhuBoGgqh%2FuO4KrrTYFqoa81uum3fuuY9pSdMtsum8LyhX6y%2F3asAFEh16a1lE5dEJ7GLtArvf7Ru2nQj7%2BMBCddpJ0YcvuWszYDmPLsHlnoA1%2Fqjc%2BX2Thy%2B45iavfseYfGKb8DH4MHc7zCmVsBjm7V1MnOS2qxZFVL4ujIZB3h7o%2BvwhSWmvYRYs1lWVqdkVWNcLW6KSV1f8ehNlbY17tyMtjezTQyT4crEq5rI%2FHXPi4kadQEFRjZ65xRbG55wW1I8aytzDR25atBjqeAWlc9Z3IvCpZC5hGoa2lo2cdvlL%2FgPPD15i09JSU7e8k1yf3pRq03fh9GwjoF6nE0N1ilz24%2FImkh2RfRG4crkDgYBsiyyMJzOmiGf8RpDMUW4NiGpqxol19fa6ZDU7caPQF35zTN%2F1wDJIZjb5r2GyTKJ9HsiQa%2Brpl1Cn3V7KqYyk%2F8yU4pSko%2Fqu0%2F7tFtZhMjv6902Dg14DKh4j1&X-Amz-Signature=0a7c70d326d706b557a443d609a7e53412a7f89666edbd12139cf7c200453642) in the `AWS CodeBuild CI Report` comment and do a `Ctrl+F` for `!!!!!!!` which will usually point you to the error.